spec/ripple/conflict/resolver_spec.rb

```diff
@@ -8,10 +8,10 @@ module Conflict
       end
 
       describe 'calling the lambda returned by .to_proc' do
-        let(:sibling_1) { stub(:data => { '_type' => 'User' } ) }
-        let(:sibling_2) { stub(:data => { '_type' => 'User' } ) }
+        let(:sibling_1) { stub(:data => { '_type' => 'User' }, :content_type => 'application/json') }
```

Comment:
Can we add this new `content-type` based test as a separate new test vs. overwriting a previous test (which implies it's not backwards compatible)?